src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformer.java

```diff
@@ -69,9 +70,9 @@
 
 public class GoGapicSurfaceTransformer implements ModelToViewTransformer {
 
-  private static final String XAPI_TEMPLATE_FILENAME = "go/main.snip";
-  private static final String SAMPLE_TEMPLATE_FILENAME = "go/example.snip";
-  private static final String DOC_TEMPLATE_FILENAME = "go/doc.snip";
+  private static final String XAPI_TEMPLATE_FILE = "go/main.snip";
```

Comment:
Remove the 'X', since that convention is gone now. Also, why are you removing the 'NAME' part? Using only 'FILE' implies the `File` class type.